content/ja/config.yaml

```diff
@@ -8,81 +8,81 @@
 #
 languageName: Japanese
```

Comment:
I think we should translate the language name as well. Testing how it looks on https://www.tensorflow.org/, it's a little unintuitive for non-native speakers if you're on the site for a language with characters. What's the right button here to get back to the English version is tricky:

<img width="928" alt="image" src="https://user-images.githubusercontent.com/98330/115954623-41eca200-a4f2-11eb-8fe5-0689cb9ee23f.png">

It's also very difficult to determine what languages are actually supported. So how about for character-based languages we change it to:
```
日本語 (Japanese)
```
?

I also thought about using country flags, but that seems to be bad practice (since multiple countries can share one language).

Reply:
> how about for character-based languages we change it to:

I think 
> 日本語 (Japanese)

is good idea.

> I also thought about using country flags, but that seems to be bad practice
Yeah. In case of Japan, I think that does not matter, but I agree we should not use country flags.

Reply:
👍🏼  for 日本語 (Japanese)